notebooks/700_metrics/701c_aupimo_advanced_ii.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<p>with the&nbsp;<code>anomalib</code>&nbsp;team from Intel's OpenVINO Toolkit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/2352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
done